source/guides/kernel/kernel-boot-msg.rst

```diff
@@ -0,0 +1,70 @@
+.. _kernel-boot-msg:
+
+Capture Kernel Boot Messages in the Journal
+###########################################
+
+By default |CL| does not capture kernel boot messages in the journal logs, 
+where they're reported as "Missed" messages. 
+
+Here's an example:
+
+.. code-block:: console
+   :linenos:
+   :emphasize-lines: 4
+
+   -- Reboot --
+   Apr 10 19:55:43 kernel systemd-journald[300]: Journal started
+   Apr 10 19:55:43 kernel systemd-journald[300]: Runtime Journal (/run/log/journal/d01862ca79d1064ea379cd715cfdd53a) is 5.8M, max 47.0M, 41.1M free.
+   Apr 10 19:55:43 kernel systemd-journald[300]: Missed 2233 kernel messages
+   Apr 10 19:55:43 kernel systemd[1]: Started Journal Service.
+
+This design decision was made to provide a faster boot performance.  
```

Comment:
So, we also need to respect to the G o o g l e monster too. Need to feed it what it wants--in the hopes that this gets indexed. That means we need to state the purpose of the doc. Right away. Move Ln 22 to follow sentence on Ln 7.  We can live without Ln 21; it's optional to include it in the intro.  And feel free to re-word current Ln 22.

Reply:
ok. done.

Reply:
Hmm. Nothing changed. Maybe check how you squashed commits.

Reply:
my bad.  i forgot to "add."  

for some reason, the automated validation process is taking much longer than usual.

Reply:
No worries. Not sure why automated validation seems to take longer.